styles/Home.module.css

```diff
@@ -165,7 +165,7 @@
   #ourserimgs div img {
     width: 200px !important;
     height: 200px !important;
-    filter: drop-shadow(0px, 0px, 10px, black) !important;
+    filter: drop-shadow(0px 0px 10px black) !important;
```

Comment:
I think both syntax are correct! please provide any docs suggesting your changes are legitimate.

Reply:
![image](https://github.com/amupedia2021/Project-Amupedia/assets/30335924/b19cf067-2436-4667-90ea-e7322d81de94)
![image](https://github.com/amupedia2021/Project-Amupedia/assets/30335924/28cf8853-72c9-4468-a06f-89b66fd61dc1)

We use commas for creating multiple shadow rather than like this.

Reply:
ok!